website/templates/website/home.html

```diff
@@ -16,6 +16,17 @@
     <header>
         <h2>Articles</h2>
     </header>
+    {% for entry in featured_entry_set %}
```

Comment:
It would be more readable to unpack this as something like `{% for entry, length featured_entry_set %}` rather than having the `0` and `1` references below.